service/src/main/java/org/folio/rest/workflow/service/OkapiDiscoveryService.java

```diff
@@ -91,7 +94,7 @@ public Map<String, List<Handler>> getHandlers(String tenant, String id) throws I
   }
 
   public JsonNode getModules(String tenant) {
-    String url = okapiUrl + PROXY_TENANT_BASE_PATH + tenant + MODULES_SUB_PATH;
+    String url = okapiUrl + basePath + PROXY_TENANT_BASE_PATH + tenant + MODULES_SUB_PATH;
```

Comment:
This is not correct. See line 33 and 34. Okapi requests to `/_/proxy/tenants/` and `/_/proxy/modules/` will be incorrect if prefexid with mod-camunda base-path anything other than `/`. 

Additional but out of scope, `okapiUrl` should have any trailing forward slash removed to be safe.